### PR TITLE
derp: update code comment for DERP phrase

### PR DESCRIPTION
**Pull request by WalterHub**

### CODE REVIEW

**Comment by bradfitz** [comment]
Why? This adds a fifth worth that doesn't fit the acronym?

**Comment by WalterHub** [comment]
There's 2 different phrases for DERP on the site and in public code.

Which of "Detour Encrypted Routing Protocol" and "Designated Encrypted Relay for Packets" should be the standard?

**Comment by bradfitz** [comment]
topic for eng meeting! :)

**Comment by bradfitz** [comment]
or recursive: DERP Encrypted Routing Protocol

File: derp/derp.go [hunk 1]
```diff
@@ -2,7 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// Package derp implements DERP, the Detour Encrypted Routing Protocol.
+// Package derp implements DERP, the Designated Encrypted Relay
```
Comment (bradfitz) [inline derp/derp.go:5]: I think the meeting ended on keeping it as either "Detour Encrypted Routing Protocol" or maybe making Detour be "Detoured". I forget.

**Review by bradfitz — CHANGES_REQUESTED** [review]

**Comment by WalterHub** [comment]
This pr is obsolete after https://github.com/tailscale/tailscale/pull/5652.

**Pull request closed without merging**